src/main/g8/default.properties

```diff
@@ -2,4 +2,4 @@ name = Hello World
 organization = com.example
 version = 1.0-SNAPSHOT
 package = $organization$.$name;format="lower,word"$
-lagom_version = maven(com.lightbend.lagom, lagom-sbt-plugin, stable)
+lagom_version = maven(com.lightbend.lagom, lagom-bom, stable)
```

Comment:
I see what you are doing here. Cleaver! 

Maybe worth to add a small comment to avoid confusion though. We us the bom to pick the version to set a `lagom_version` variable that's used to set the plugin version.

Reply:
I added a small comment.

Reply:
Just saw it. I fixed a small typo and committed already.